<launch>
  <node pkg="navigator_shoot_vision" type="shape_finder" name="dock_shape_finder">
    <rosparam file="$(find navigator_shoot_vision)/launch/shape_finder.yaml" />
    <param name="symbol_camera" value="/right/right/image_raw"/>
    <param name="image_transport" value="raw" />
    <param name="auto_start" value="false" />
  </node>
  <!-- node pkg="navigator_shoot_vision" type="shape_filter" name="dock_shape_filter">
    <rosparam file="$(find navigator_shoot_vision)/launch/shape_filter.yaml" />
    <param name="auto_start" value="false" />
  </node -->
  <node pkg="navigator_shoot_vision" type="ray_lidar" name="camera_lidar_transformer">
    <rosparam file="$(find navigator_shoot_vision)/launch/lidar_transformer.yaml" />
    <param name="camera_info_topic" value="/right/right/camera_info" />
    <param name="auto_start" value="false" />
  </node>
</launch>
